docs/examples/plot_backtracking_sloped_terrain.py

```diff
@@ -0,0 +1,168 @@
+#
+# The sign convention for ground slope follows the same convention -- align
+# the right-hand thumb along the tracker axis in the direction of the axis
+# azimuth and the fingers curl towards positive angles. So in this example,
+# with the axis azimuth coming out of the page, an east-facing slope is a
+# negative rotation from horizontal:
+#
+# .. image:: ../_images/ground_slope_angle_convention.png
+#   :alt: Image showing the ground slope sign convention.
+#   :width: 500
+#   :align: center
+#
+
+# %%
+# Rotation curves
+# ---------------
+#
+# Now, let's plot the simple case where the tracker axes are at right angles
+# to the direction of the slope.  In this case, the cross-axis tilt angle
+# is the same as the slope of the terrain and the tracker axis itself is
+# horizontal.
+
+from pvlib import solarposition, tracking
+import pandas as pd
+import matplotlib.pyplot as plt
+
+# PV system parameters
+tz = 'US/Eastern'
+lat, lon = 40, -80
+gcr = 0.4
+
+# calculate the solar position
+times = pd.date_range('2019-01-01 06:00', '2019-01-01 18:00', closed='left',
+                      freq='1min', tz=tz)
+solpos = solarposition.get_solarposition(times, lat, lon)
+
+# compare the backtracking angle at various terrain slopes
+fig, ax = plt.subplots()
+for cross_axis_tilt in [0, 5, 10]:
+    tracker_data = tracking.singleaxis(
+        apparent_zenith=solpos['apparent_zenith'],
+        apparent_azimuth=solpos['azimuth'],
+        axis_tilt=0,  # flat because the axis is perpendicular to the slope
+        axis_azimuth=180,  # N-S axis, azimuth facing south
+        max_angle=90,
+        backtrack=True,
+        gcr=gcr,
+        cross_axis_tilt=cross_axis_tilt)
+
+    backtracking_position = tracker_data['tracker_theta'].fillna(0)
+    label = 'cross-axis tilt: {}°'.format(cross_axis_tilt)
+    backtracking_position.plot(label=label, ax=ax)
+
+plt.legend()
+plt.title('Backtracking Curves')
+plt.show()
+
+# %%
+# This plot shows how backtracking changes based on the slope between rows.
+# For example, unlike the flat-terrain backtracking curve, the sloped-terrain
+# curves do not approach zero at the end of the day. Because of the vertical
+# offset between rows introduced by the sloped terrain, the trackers can be
+# slightly tilted without shading each other.
+#
+# Now let's examine the general case where the terrain slope makes an
+# inconvenient angle to the tracker axes. For example, consider an array
+# with north-south axes on terrain that slopes down to the south-south-east.
+# Assuming the axes are installed parallel to the ground, the northern ends
+# of the axes will be higher than the southern ends. But because the slope
+# isn't purely parallel or perpendicular to the axes, the axis tilt and
+# cross-axis tilt angles are not immediately obvious. We can use pvlib
+# to calculate them for us:
+
+slope_azimuth = 155  # terrain slopes down to the south-south-east
+slope_tilt = 10  # terrain is sloped at 10 degrees from horizontal
```

Comment:
I'm confused at this point. The slope direction is 155 azimuth, but, the tilt is 10 degrees relative to what? Is it 10 degrees rotation relative to the tracker azimuth of 180? That seems an odd choice, since it is difficult to measure the slope tilt except relative to the slope azimuth.

If slope_tilt is 10 degrees relative to horizontal along slope_azimuth, then I'd suggest

```suggestion
slope_tilt = 10  # terrain slopes upward 10 degrees from horizontal in the direction of slope_azimuth
```

Reply:
> If slope_tilt is 10 degrees relative to horizontal along slope_azimuth,

Correct, but I think your suggested comment has the wrong sign.  It's either upwards 10 degrees away from slope_azimuth, or downward 10 degrees towards it.  Are you okay with this revision?

> terrain slopes upward 10 degrees from horizontal away from the direction of slope_azimuth

Reply:
slope_azimuth is chosen in the downslope direction? I may have missed that if it's earlier in the text. If not, would be good to add. If slope_azimuth is always downslope, then I think it may be more clear to describe slope_tilt in the direction of slope_azimuth, i.e., always negative.

Reply:
Thanks for your patience with this reviewer

Reply:
Changes made.  As always, I am grateful for the review :)